From 840e0d84469210953af223746a9370036d404d38 Mon Sep 17 00:00:00 2001
From: "Kris Thielemans@ucl.ac.uk" <k.thielemans@ucl.ac.uk>
Date: Thu, 23 Jul 2026 09:22:20 +0100
Subject: [PATCH] optionally as curl dependency

---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a5ffda9..ecdc52c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,6 +183,12 @@ if (USE_HDF5_DATASET_SUPPORT)
     set(ISMRMRD_DATASET_SOURCES libsrc/dataset.c libsrc/dataset.cpp)
     message(STATUS "HDF5 include found at: ${HDF5_INCLUDE_DIRS}")
     message(STATUS "HDF5 libs found at: ${HDF5_C_LIBRARIES}")
+    find_package(CURL QUIET)
+    if (CURL_FOUND)
+        message(STATUS "Adding Curl to HDF5 dependencies to cover FindHDF5.cmake deficiencies")
+        set(ISMRMRD_DATASET_INCLUDE_DIR ${CURL_INCLUDE_DIRS})
+        set(ISMRMRD_DATASET_LIBRARIES ${ISMRMRD_DATASET_LIBRARIES} ${CURL_LIBRARIES})
+    endif()
 else ()
     set(ISMRMRD_DATASET_SUPPORT false)
     message(WARNING " Dataset and file support unavailable!")
-- 
2.43.0

