diff --git a/CMakeLists.txt b/CMakeLists.txt
index f78e1f0..382444f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,7 +78,7 @@ else()
    message("Building statically linked application.")
 endif()
 
-find_package(Boost COMPONENTS system program_options filesystem REQUIRED)
+find_package(Boost COMPONENTS program_options filesystem REQUIRED)
 find_package(ISMRMRD 1.14.2 REQUIRED)
 
 add_executable(embed base64.cpp embed.cpp)
